docs/component/oa-verify.md

```diff
@@ -119,61 +76,38 @@ The `isValid` function will execute over fragments and determine if the fragment
 
-The function allow to specify as a second parameters the list of types on which to perform the checks. Let's try it with our custom verifiers over the initial wrapped document:
+The function allow to specify as a second parameters the list of types on which to perform the checks. Let's try to run the verifier on `mainnet` network :
 
 ```javascript
-const { verificationBuilder, isValid, openAttestationVerifiers } = require("@govtechsg/oa-verify");
+const { verify, isValid } = require("@govtechsg/oa-verify");
 const document = require("./document.json");
 
-// our custom verifier will be valid only if the document version is not open-attestation/2.0
-const customVerifier = {
-  skip: () => {
-    throw new Error("this verifier is never skipped");
-  },
-  test: () => true,
-  verify: async document => {
-    if (document.version === "open-attestation/2.0") {
-      return {
-        type: "DOCUMENT_INTEGRITY",
-        name: "CustomVerifier",
-        data: document.version,
-        status: "INVALID"
-      };
-    }
-    return {
-      type: "DOCUMENT_INTEGRITY",
-      name: "CustomVerifier",
-      data: document.version,
-      status: "VALID"
-    };
-  }
-};
-
-const verify = verificationBuilder([
-  ...openAttestationVerifiers,
-  customVerifier // see above
-]);
-
-verify(document, { network: "ropsten" }).then(fragments => {
+verify(document, { network: "mainnet" }).then(fragments => {
   console.log(isValid(fragments)); // output false
-  console.log(isValid(fragments, ["DOCUMENT_INTEGRITY"])); // output false
-  console.log(isValid(fragments, ["DOCUMENT_STATUS"])); // output true
-  console.log(isValid(fragments, ["ISSUER_IDENTITY"])); // output true
+  console.log(isValid(fragments, ["DOCUMENT_INTEGRITY"])); // output true
+  console.log(isValid(fragments, ["DOCUMENT_STATUS"])); // output false
+  console.log(isValid(fragments, ["ISSUER_IDENTITY"])); // output false
 });
 ```
 
-### Listening to individual verifiers
-The `verify` function has an option to listen to every verifiers individually. It might be useful if you want for instance to provide individual loader on your UI.
+Let's try to understand the different results:
+- `isValid(fragments, ["DOCUMENT_INTEGRITY"])` returns true because the integrity of the document is not dependant on the network it has been published to.
+- `isValid(fragments, ["DOCUMENT_STATUS"])` returns false because the document has been published on ropsten network only.
+- `isValid(fragments, ["DOCUMENT_STATUS"])` returns false because the [DNS-TXT record](/docs/verifiable-document/dns-proof.md) associated with the document is only available for rospten network.
+- `isValid(fragments)` returns false because at least one of the above returns false.
+
+### Listening to individual verification method
+The `verify` function has an option to listen to every verification method individually. It might be useful if you want for instance to provide individual loader on your UI.
```

Comment:
```suggestion
The `verify` function has an option to listen to individual verification methods. It might be useful if you want for instance to provide individual loader on your UI.
```